pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: dpx
-version: 0.1.0
+version: 0.1.1
```

Comment:
#nit, I noticed that [`version.dart`](https://github.com/Workiva/dpx/blob/master/lib/src/version.dart) isn't getting updated

meaning `dpx --version` is outputting an incorrect version

Not blocking, but is a bit confusing especially given the issue template asking to run `dpx --version`

Reply:
I totally forgot about that :) I'll fix that up in a separate PR.